src/daniil_efremov_ex/ex01/letter_counts.py

```diff
@@ -0,0 +1,13 @@
+def letter_freq(txt):
+    txt.lower()
```

Comment:
This line doesn't change `txt` to lower-cased, you must assign it to a variable to get the all-lower-cased text.
You can try run the file again with "Hello, Jim".